Source/SwiftLintFramework/Rules/RuleConfigurations/NameConfiguration.swift

```diff
@@ -17,6 +17,8 @@ public struct NameConfiguration: RuleConfiguration, Equatable {
     var minLength: SeverityLevelsConfiguration
     var maxLength: SeverityLevelsConfiguration
     var excluded: Set<String>
+    var allowedSymbols: Set<String>
```

Comment:
this configuration is used on `TypeNameRule` and `GenericTypeNameRule` too. Maybe we should add support on those rules as well?